src/alignTransform.js

```diff
@@ -101,6 +101,10 @@ const alignTransform = ({
       }
     }
 
+    if (tokens.tag === '' && (tokens.type || tokens.name)) {
```

Comment:
```suggestion
    if (tokens.tag === '' && tokens.type) {
```

Don't apply this if it is needed, but I think you will not get coverage for `tokens.name` because there will always be a type if there is a multiline name. `comment-parser` seems to treat the following as having only a description (i.e., `name desc`):

```js
/**
 * @tag
 * name desc
 */
```

If I am wrong, and the extra check is needed, however, please modify my to-do comment above to indicate that `name` is also checked.